package.json

```diff
@@ -29,13 +29,19 @@
   "dependencies": {
     "@nestjs/bull": "^0.3.1",
     "@nestjs/common": "^7.0.0",
+    "@nestjs/config": "^1.0.1",
     "@nestjs/core": "^7.0.0",
+    "@nestjs/passport": "^8.0.1",
     "@nestjs/platform-express": "^7.0.0",
     "@nestjs/platform-ws": "^7.6.18",
     "@nestjs/websockets": "^7.6.18",
     "@types/bull": "^3.15.1",
     "bull": "^3.22.9",
+    "express": "^4.17.1",
+    "express-session": "^1.17.2",
     "nest-commander": "^1.2.0",
+    "passport": "^0.4.1",
+    "passport-twitter": "^1.0.4",
```

Comment:
Can we make sure to add the `@types/` packages for express, express-session, passport, and passport-twitter?